installation/scripts/prow/provision.sh

```diff
@@ -41,3 +41,22 @@ else
     sudo ${INSTALLATION_DIR}/cmd/run.sh --k3d-memory 12288MB
 fi
 
+# Patch deployment imagePullPolicy to always pull image on subsequent executions of update-compass.sh
```

Comment:
It would be faster from prow job PoV to do that patching on the charts level with some cli text editor like sed on the charts.